### PR TITLE
add pytest-shutil recipe

### PR DESCRIPTION
**Pull request by jcfr**
See https://github.com/manahl/pytest-plugins/tree/master/pytest-shutil#readme

### CODE REVIEW

File: pytest-shutil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  imports:
+    - pytest_shutil
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/manahl/pytest-plugins
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (marcelotrevisani) [inline pytest-shutil/meta.yaml]: Please change that to
```
  license_file: 'LICENSE'
```
The license file is already being downloaded, so, it's not necessary to upload that as well

File: pytest-shutil/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (marcelotrevisani) [inline pytest-shutil/LICENSE]: Please, remove this file

**Comment by jcfr** [comment]
Thanks for the review :+1: 

We should be all set.

To provide some background ....

> python -m pip install --no-deps --ignore-installed .

I went with the the value that `conda skeleton pypi pytest-shutil` gave me, I wonder we should update the skeleton generator.

> pytest-shutil/LICENSE

I inspired from what was done in https://github.com/conda-forge/staged-recipes/pull/6328

File: pytest-shutil/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2cfe4d3f5f25ad2b19e64847d62563f5494b2e0450ca1cfc5940974029b2cbd1
+
+build:
+  number: 0
+  script: "python -m pip install --no-deps --ignore-installed ."
```
Comment (marcelotrevisani) [inline pytest-shutil/meta.yaml]: Please remove the " from this part

**Comment by marcelotrevisani** [comment]
Please put your folder inside of the ``recipes`` folder

**Comment by marcelotrevisani** [comment]
I already did that, just a minor change to your recipe be in the right folder

File: recipes/pytest-shutil/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    - six
+    - termcolor
+  run:
+    - contextlib2
+    - execnet
+    - mock
+    - path.py
+    - pytest
+    - python
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/pytest-shutil/meta.yaml]: Please, remove setuptools from here

File: recipes/pytest-shutil/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    - contextlib2
+    - execnet
+    - mock
+    - path.py
+    - pip
+    - pytest
+    - python
+    - setuptools
+    - six
+    - termcolor
```
Comment (marcelotrevisani) [inline recipes/pytest-shutil/meta.yaml]: Please add ``setuptools-git``

**Comment by marcelotrevisani** [comment]
Now everything should be fine. Waiting the ``appveyor`` to finish and I will merge that

**Comment by jcfr** [comment]
Thanks again. I will make sure to follow these directions for the next package I create.

**Pull request merged**